schemas/context/viewability-details.schema.json

```diff
@@ -7,18 +7,17 @@
   ],
   "$id": "https://ns.adobe.com/xdm/context/viewability-details",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "Viewability",
+  "title": "Viewability Details",
   "type": "object",
-  "description":
-  "Description of viewability measured properties of an advertisement",
+  "description": "Browser or app settings measured at runtime describing the viewability of an advertisement or content associated with each page load.",
   "definitions": {
     "viewabilityDetails": {
```

Comment:
this should be `viewability-details`

Reply:
Thanks for pointing this out, also corrected allOf node